lms/templates/student_account/register.underscore

```diff
@@ -1,6 +1,8 @@
-<div class="status submission-error hidden" aria-live="polite">
+<div class="status submission-error <% if (!context.ipAddressWarningMsg){ %>hidden<% } %>" aria-live="polite">
```

Comment:
it looks like you're trying to solve the same issue with duplication but differently. I need a call to see how it works.